src/app/backend/resource/pod/detail.go

```diff
@@ -165,14 +158,8 @@ func GetPodDetail(client kubernetes.Interface, metricClient metricapi.MetricClie
 	return &podDetail, nil
 }
 
-func getPodCreator(client kubernetes.Interface, creatorAnnotation string, nsQuery *common.NamespaceQuery,
-	pod *v1.Pod) (*controller.ResourceOwner, error) {
-
-	var serializedReference v1.SerializedReference
-	err := json.Unmarshal([]byte(creatorAnnotation), &serializedReference)
-	if err != nil {
-		return nil, err
-	}
+func getPodCreator(client kubernetes.Interface, nsQuery *common.NamespaceQuery, pod *v1.Pod) (
```

Comment:
Will `getPodController` be a better name? The controller that the function returns may not be the creator (the parent may change).